internal/appsec/listener/waf/tags.go

```diff
@@ -83,19 +73,5 @@ func SetEventSpanTags(span trace.TagSetter, events []any) error {
 	span.SetTag("_dd.origin", "appsec")
 	// Set the appsec.event tag needed by the appsec backend
 	span.SetTag("appsec.event", true)
-	return nil
-}
-
-// Create the value of the security event tag.
-func makeEventTagValue(events []any) (json.RawMessage, error) {
-	type eventTagValue struct {
-		Triggers []any `json:"triggers"`
-	}
-
-	tag, err := json.Marshal(eventTagValue{events})
-	if err != nil {
-		return nil, fmt.Errorf("unexpected error while serializing the appsec event span tag: %v", err)
-	}
-
-	return tag, nil
+	span.SetTag("_dd.p.appsec", internal.PropagatingTagValue{Value: "1"})
```

Comment:
I wonder if we should put the Force Keep tagging over here to have everything related all together.

Reply:
The diff is broken but it's here, 5 lines atop of this one

Reply:
```go
// SetEventSpanTags sets the security event span tags related to an appsec event
func SetEventSpanTags(span trace.TagSetter) {
	span.SetTag(ext.ManualKeep, samplernames.AppSec)
	span.SetTag("_dd.origin", "appsec")
	// Set the appsec.event tag needed by the appsec backend
	span.SetTag("appsec.event", true)
	span.SetTag("_dd.p.appsec", internal.PropagatingTagValue{Value: "1"})
}
```

Reply:
It's not that it's broken, it's just out-of-context-window 🙃

That said (optional) -- you could order the statements so these things are closer together.